### ISSUE TITLE
FEATURE REQUEST: Parent/ Child view on front of card w/ drag n' drop ability

### ISSUE THREAD

**Issue by oneezy** [handsomecode/trello-parent-child#1]
ɪᴛ'ᴅ ʙᴇ ᴍɪɢʜᴛʏ ʜᴀɴᴅsᴏᴍ⋈ᴇ 
:)

![image](https://cloud.githubusercontent.com/assets/7369575/12519305/b9ed0ac0-c103-11e5-95d8-94b4f3968b27.png)

**Comment by alexzub** [comment]
@oneezy that's a great idea Justin. Adding to our backlog!
(feel free to fork and code it yourself – the plugin is totally open source!)

**Comment by ghost** [comment]
Great idea!! +1

**Comment by oneezy** [comment]
Thanks guys!

**Comment by oneezy** [comment]
@azmorf , I might attempt that. "UI Guy" over here... hahA.. I'm getting braver and braver each day though.

**Comment by oneezy** [comment]
@azmorf @valerio1 ,  have you guys attempted this at all? I'd be willing to help!

**Comment by alexzub** [comment]
@oneezy Nope, haven't had a chance to focus on the plugin in a while now unfortunately. You're welcome to create a fork; we'll make sure to do a code review from our side to make sure everything aligns with the current code of the plugin!

**Comment by oneezy** [comment]
Right on, I'll grab the code here in a bit and have a look.

If you guys have a public Trello dev board, shoot me over a link. If not, I'll create one and lay out a plan.
